azkaban-common/src/test/java/azkaban/trigger/BasicTimeCheckerTest.java

```diff
@@ -109,14 +107,18 @@ public void cronTimerTest1() {
   /**
    * Test when PST-->PDT happens in 2020. -8:00 -> -7:00
    * See details why confusion happens during this change: https://en.wikipedia.org/wiki/Pacific_Time_Zone
+   *
+   * This test demonstrates that
```

Comment:
Clarify that this test is designed to show that using UTC timezone will not skip a run but will shift execution time  based on the Pacific time when DST changes.